### ISSUE TITLE
Voice recognition not working

### ISSUE THREAD

**Issue by kengoon** [AM-ash-OR-AM-I/Find_MyStuff#1]
Why is voice recognition not working

**Comment by AshuK8s** [comment]
> Why is voice recognition not working

@kengoon Can you please describe the issue? Like Android version you are using? Camera is working or not?

**Comment by kengoon** [comment]
@AM-ash-OR-AM-I Yes I am using the android version, Camera is working perfect, just the voice recognition stuff

**Comment by AshuK8s** [comment]
> @AM-ash-OR-AM-I Yes I am using the android version, Camera is working perfect, just the voice recognition stuff

@kengoon I meant what is the Android version that you are using? Is it Android Lollipop? Because I've tested it on all devices from Android 7 Nougat to Android 11 and it works alright... I think issue might be with older Android version like lollipop and marshmallow..

**Comment by kengoon** [comment]
> > @AM-ash-OR-AM-I Yes I am using the android version, Camera is working perfect, just the voice recognition stuff
> 
> @kengoon I meant what is the Android version that you are using? Is it Android Lollipop? Because I've tested it on all devices from Android 7 Nougat to Android 11 and it works alright... I think issue might be with older Android version like lollipop and marshmallow..

I use Android 7

**Comment by AshuK8s** [comment]
> > > @AM-ash-OR-AM-I Yes I am using the android version, Camera is working perfect, just the voice recognition stuff
> > 
> > 
> > @kengoon I meant what is the Android version that you are using? Is it Android Lollipop? Because I've tested it on all devices from Android 7 Nougat to Android 11 and it works alright... I think issue might be with older Android version like lollipop and marshmallow..
> 
> I use Android 7

Did you allow microphone permission in the dialog box? Check if microphone permission is enabled for the application by going to app info. If it is enabled then I think something else is going wrong because it works alright in Android 7 devices that I've tested.
 
P.S. What language do you use for speech to text in your phone? If you don't use English then that might be issue.

**Comment by AshuK8s** [comment]
Try using this older version and see if this works... https://github.com/AM-ash-OR-AM-I/Find_MyStuff/releases/download/v0.1/com.FindMyStuff-0.1-armeabi-v7a-debug.apk

**Comment by kengoon** [comment]
@AM-ash-OR-AM-I I allowed the mic permission and also my phone is English

**Comment by kengoon** [comment]
@AM-ash-OR-AM-I also from the video I saw on the repo(not youtube video), it looks different from the one I have, maybe you have to update the apk

**Comment by AshuK8s** [comment]
> @AM-ash-OR-AM-I also from the video I saw on the repo(not youtube video), it looks different from the one I have, maybe you have to update the apk

@kengoon Yes I had updated the apk with new UI and fixed some of the earlier issues that it had, but I didn't have time to change the demo video, the video that you saw was of old version. Now I've changed the video.

Check if this older version works...
https://github.com/AM-ash-OR-AM-I/Find_MyStuff/releases/download/v0.1/com.FindMyStuff-0.1-armeabi-v7a-debug.apk

Also I'd appreciate if  you can do screen recording of the mic not working, I think what must be happening is that the animation is of mic must be showing but it isn't recognizing anything right? I had this issue in android 11 but I had fixed that in new version and I didn't face any issue in Android 7....

**Comment by kengoon** [comment]
> > @AM-ash-OR-AM-I also from the video I saw on the repo(not youtube video), it looks different from the one I have, maybe you have to update the apk
> 
> @kengoon Yes I'd updated the apk with new UI and fixed some of the earlier issues that it had.... Check if this older version works 
> https://github.com/AM-ash-OR-AM-I/Find_MyStuff/releases/download/v0.1/com.FindMyStuff-0.1-armeabi-v7a-debug.apk

@AM-ash-OR-AM-I It is still not working

**Comment by AshuK8s** [comment]
@kengoon Can you screen record the application?

**Comment by kengoon** [comment]
@AM-ash-OR-AM-I  Sorry for late reply, I was really busy. I will do as you requested and give you feedback

**Comment by AshuK8s** [comment]
Sure I have also made a new version of this app that fixed some more issues
with mic but still not sure whether it'll work on your phone. I'll be a bit
busy, today I'll see your recording and publish this new version soon when
I'll have some time.

**Comment by AshuK8s** [comment]
> 
> 
> @AM-ash-OR-AM-I Sorry for late reply, I was really busy. I will do as you requested and give you feedback

@kengoon Now, I've updated apk and code, could you download again and see if it's working alright or not?

**Comment by kengoon** [comment]
@AM-ash-OR-AM-I  Ok 👌

**Comment by kengoon** [comment]
It is still not working, below is a video record that I did
https://user-images.githubusercontent.com/42192162/132991916-80b737eb-429b-4c3f-98f6-015f2348714a.mp4

**Comment by AshuK8s** [comment]
> 
> 
> It is still not working, below is a video record that I did
> https://user-images.githubusercontent.com/42192162/132991916-80b737eb-429b-4c3f-98f6-015f2348714a.mp4

Ok bro got it, I'll look for some solutions.
